tests/posix/threads_ext/src/main.c

```diff
@@ -446,7 +446,7 @@ ZTEST(pthread_attr, test_pthread_attr_setstacksize)
 	}
 }
 
-ZTEST(pthread_attr, test_pthread_attr_getscope)
+ZTEST(posix_threads_ext, test_pthread_attr_getscope)
```

Comment:
I believe this is part of POSIX_THREADS_BASE and likely not POSIX_THREADS_EXT

But it does require the CONFIG_POSIX_THREAD_PRIORITY_SCHEDULING option.